community/libmustache/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Renoir Boulanger <hello@renoirboulanger.com>
+# Maintainer: John Boehr <jbboehr@gmail.com>
+pkgname=libmustache
+pkgver=0.4.2
+pkgrel=1
+pkgdesc="C++ implementation of Mustache intended mainly for use as a PHP extension"
+url="https://github.com/jbboehr/libmustache"
+arch="all"
+license="MIT"
+depends="libgcc libstdc++"
+makedepends="autoconf automake libtool git"
+subpackages=""
+source="$pkgname-$pkgver.tar.gz::https://github.com/jbboehr/$pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+prepare() {
+	cd "$builddir"
+	git clone https://github.com/jbboehr/mustache-spec.git spec && \
+	cd spec && \
+	git reset --hard b96be9fd4c6d6984828d93169fe7e86d8a8aec2f && \
+	cd ..
+	autoreconf -vif
+	default_prepare || return 1
+}
+
+build() {
+	cd "$builddir"
+	./configure \
+		--prefix=/usr \
+		|| return 1
+	make || return 1
+}
+
```

Comment:
Typically, it's meant as a `make check` runner -- performing tests on whether the package functions successfully, so kind of like integration tests. We usually don't mind it running unit tests and the like as well as long as they don't take too long.